#! /bin/sh -e
## 06_dautipps.dpatch by Eduard Bloch <blade@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Various small fixes

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urd rxvt-unicode-3.9/doc/rxvt.1.man rxvt-unicode-3.9.my/doc/rxvt.1.man
--- rxvt-unicode-3.9/doc/rxvt.1.man	2004-08-25 04:34:38.000000000 +0200
+++ rxvt-unicode-3.9.my/doc/rxvt.1.man	2004-08-28 11:11:54.000000000 +0200
@@ -129,12 +129,12 @@
 .\" ========================================================================
 .\"
 .IX Title "rxvt 1"
-.TH rxvt 1 "2004-08-25" "perl v5.8.4" "RXVT-UNICODE"
+.TH rxvt 1 "2004-08-28" "perl v5.8.4" "RXVT-UNICODE"
 .SH "NAME"
 rxvt\-unicode (ouR XVT, unicode) \- (a VT102 emulator for the X window system)
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
-\&\fBrxvt\fR [options] [\-e command [ args ]]
+\&\fBurxvt\fR [options] [\-e command [ args ]]
 .SH "DESCRIPTION"
 .IX Header "DESCRIPTION"
 \&\fBrxvt-unicode\fR, version \fB3.8\fR, is a colour vt102 terminal
@@ -154,7 +154,7 @@
 fine, though. A somewhat difficult case are left-to-right scripts, such
 as hebrew: \fBrxvt-unicode\fR adopts the view that bidirectional algorithms
 belong into the application, not the terminal emulator (too many things \*(--
-such as cursor-movement while editing \*(-- break othwerwise), but that might
+such as cursor-movement while editing \*(-- break otherwise), but that might
 change.
 .PP
 If you are looking for a terminal that supports more exotic scripts, let
@@ -178,28 +178,28 @@
 without most of it's features to get a lean binary. It also comes with
 a client/daemon pair that lets you open any number of terminal windows
 from within a single process, which makes startup time very fast and
-drastically reduces memory usage. See \fIrxvtd\fR\|(1) (daemon) and
-\&\fIrxvtc\fR\|(1) (client).
+drastically reduces memory usage. See \fIurxvtd\fR\|(1) (daemon) and
+\&\fIurxvtc\fR\|(1) (client).
 .PP
 It also makes technical information about escape sequences (which have
-been extended) easier accessible: see \fIrxvt\fR\|(7) for technical
+been extended) easier accessible: see \fIurxvt\fR\|(7) for technical
 reference documentation (escape sequences etc.) and the \s-1FAQ\s0 section at the
 end of this document.
 .SH "OPTIONS"
 .IX Header "OPTIONS"
-The \fBrxvt\fR options (mostly a subset of \fIxterm\fR's) are listed
+The \fBurxvt\fR options (mostly a subset of \fIxterm\fR's) are listed
 below. In keeping with the smaller-is-better philosophy, options may be
 eliminated or default values chosen at compile\-time, so options and
 defaults listed may not accurately reflect the version installed on
-your system. `rxvt \-h' gives a list of major compile-time options on
+your system. `urxvt \-h' gives a list of major compile-time options on
 the \fIOptions\fR line. Option descriptions may be prefixed with which
 compile option each is dependent upon. e.g. `Compile \fI\s-1XIM\s0\fR:' requires
-\&\fI\s-1XIM\s0\fR on the \fIOptions\fR line. Note: `rxvt \-help' gives a list of all
+\&\fI\s-1XIM\s0\fR on the \fIOptions\fR line. Note: `urxvt \-help' gives a list of all
 command-line options compiled into your version.
 .PP
-Note that \fBrxvt\fR permits the resource name to be used as a
+Note that \fBurxvt\fR permits the resource name to be used as a
 long-option (\-\-/++ option) so the potential command-line options are
-far greater than those listed. For example: `rxvt \-\-loginShell \-\-color1
+far greater than those listed. For example: `urxvt \-\-loginShell \-\-color1
 Orange'.
 .PP
 The following options are available:
@@ -361,7 +361,7 @@
 resource \fBtermName\fR.
 .IP "\fB\-e\fR \fIcommand [arguments]\fR" 4
 .IX Item "-e command [arguments]"
-Run the command with its command-line arguments in the \fBrxvt\fR
+Run the command with its command-line arguments in the \fBurxvt\fR
 window; also sets the window title and icon name to be the basename of
 the program being executed if neither \fI\-title\fR (\fI\-T\fR) nor \fI\-n\fR are
 given on the command line. If this option is used, it must be the last
@@ -419,27 +419,27 @@
 some window managers.
 .SH "RESOURCES (available also as long\-options)"
 .IX Header "RESOURCES (available also as long-options)"
-Note: `rxvt \-\-help' gives a list of all resources (long
+Note: `urxvt \-\-help' gives a list of all resources (long
 options) compiled into your version.
 .PP
-There are two different methods that rxvt can use to get the
+There are two different methods that urxvt can use to get the
 Xresource data: using the X libraries (Xrm*\-functions) or internal
 Xresources reader (\fB~/.Xdefaults\fR). For the first method (ie.
-\&\fBrxvt \-h\fR lists \fBXGetDefaults\fR), you can set and change the
+\&\fBurxvt \-h\fR lists \fBXGetDefaults\fR), you can set and change the
 resources using X11 tools like \fBxset\fR. Many distribution do also load
 settings from the \fB~/.Xresources\fR file when X starts.
 .PP
-If compiled with internal Xresources support (i.e. \fBrxvt \-h\fR
-lists \fB.Xdefaults\fR) then \fBrxvt\fR accepts application defaults
+If compiled with internal Xresources support (i.e. \fBurxvt \-h\fR
+lists \fB.Xdefaults\fR) then \fBurxvt\fR accepts application defaults
 set in XAPPLOADDIR/URxvt (compile\-time defined: usually
 \&\fB/usr/lib/X11/app\-defaults/URxvt\fR) and resources set in
 \&\fB~/.Xdefaults\fR, or \fB~/.Xresources\fR if \fB~/.Xdefaults\fR does not exist.
-Note that when reading X resources, \fBrxvt\fR recognizes two
+Note that when reading X resources, \fBurxvt\fR recognizes two
 class names: \fBXTerm\fR and \fBURxvt\fR. The class name \fBRxvt\fR allows
-resources common to both \fBrxvt\fR and the original \fIrxvt\fR to be
+resources common to both \fBurxvt\fR and the original \fIrxvt\fR to be
 easily configured, while the class name \fBURxvt\fR allows resources
-unique to \fBrxvt\fR, notably colours and key\-handling, to be
-shared between different \fBrxvt\fR configurations. If no
+unique to \fBurxvt\fR, notably colours and key\-handling, to be
+shared between different \fBurxvt\fR configurations. If no
 resources are specified, suitable defaults will be used. Command-line
 arguments can be used to override resource settings. The following
 resources are allowed:
@@ -465,7 +465,7 @@
 names used are listed in the \fB\s-1COLORS\s0 \s-1AND\s0 \s-1GRAPHICS\s0\fR section.
 .Sp
 Colours higher than 15 cannot be set using resources (yet), but can be
-changed using an escape command (see \fIrxvt\fR\|(7)).
+changed using an escape command (see \fIurxvt\fR\|(7)).
 .Sp
 Colours 16\-79 form a standard 4x4x4 colour cube (the same as xterm with
 88 colour support). Colours 80\-87 are evenly spaces grey steps.
@@ -684,7 +684,7 @@
 \&\fB+si\fR.
 .IP "\fBscrollWithBuffer:\fR \fIboolean\fR" 4
 .IX Item "scrollWithBuffer: boolean"
-\&\fBTrue\fR: scroll with scrollback buffer when tty recieves new lines (and
+\&\fBTrue\fR: scroll with scrollback buffer when tty receives new lines (and
 \&\fBscrollTtyOutput\fR is False); option \fB+sw\fR. \fBFalse\fR: do not scroll
 with scrollback buffer when tty recieves new lines; option \fB\-sw\fR.
 .IP "\fBscrollTtyKeypress:\fR \fIboolean\fR" 4
@@ -776,7 +776,7 @@
 .IX Item "insecure: boolean"
 Enables \*(L"insecure\*(R" mode. Rxvt-unicode offers some escape sequences that
 echo arbitrary strings like the icon name or the locale. This could be
-abused if somebody gets 8\-bit\-clean access to your display, wether
+abused if somebody gets 8\-bit\-clean access to your display, whether
 throuh a mail client displaying mail bodies unfiltered or though
 \&\fIwrite\fR\|(1). Therefore, these sequences are disabled by default. (Note
 that other terminals, including xterm, have these sequences
@@ -815,9 +815,9 @@
 \&\s-1KEYSYM_RESOURCE\s0.
 .SH "THE SCROLLBAR"
 .IX Header "THE SCROLLBAR"
-Lines of text that scroll off the top of the \fBrxvt\fR window
+Lines of text that scroll off the top of the \fBurxvt\fR window
 (resource: \fBsaveLines\fR) and can be scrolled back using the scrollbar
-or by keystrokes. The normal \fBrxvt\fR scrollbar has arrows and
+or by keystrokes. The normal \fBurxvt\fR scrollbar has arrows and
 its behaviour is fairly intuitive. The \fBxterm-scrollbar\fR is without
 arrows and its behaviour mimics that of \fIxterm\fR
 .PP
@@ -849,7 +849,7 @@
 .IP "\fBInsertion\fR:" 4
 .IX Item "Insertion:"
 Pressing and releasing the Middle mouse button (or \fBShift-Insert\fR) in
-an \fBrxvt\fR window causes the current text selection to be
+an \fBurxvt\fR window causes the current text selection to be
 inserted as if it had been typed on the keyboard.
 .SH "CHANGING FONTS"
 .IX Header "CHANGING FONTS"
@@ -897,7 +897,7 @@
 invoke it's usual function but instead will insert the corresponding
 keycap symbol. The symbol will only be entered when the key has been
 released, otherwise pressing e.g. \f(CW\*(C`Shift\*(C'\fR would enter the symbol for
-\&\f(CW\*(C`ISO Level 2 Switch\*(C'\fR, although your intention might have beenm to enter a
+\&\f(CW\*(C`ISO Level 2 Switch\*(C'\fR, although your intention might have been to enter a
 reverse tab (Shift\-Tab).
 .IP "5.3: Screen-selection entry method" 4
 .IX Item "5.3: Screen-selection entry method"
@@ -922,14 +922,14 @@
 both scenario A and B of \s-1ISO\s0 14755, including part 5.2.
 .SH "LOGIN STAMP"
 .IX Header "LOGIN STAMP"
-\&\fBrxvt\fR tries to write an entry into the \fIutmp\fR(5) file so
+\&\fBurxvt\fR tries to write an entry into the \fIutmp\fR(5) file so
 that it can be seen via the \fI\fIwho\fI\|(1)\fR command, and can accept messages.
-To allow this feature, \fBrxvt\fR must be installed setuid root on
+To allow this feature, \fBurxvt\fR must be installed setuid root on
 some systems.
 .SH "COLORS AND GRAPHICS"
 .IX Header "COLORS AND GRAPHICS"
 In addition to the default foreground and background colours,
-\&\fBrxvt\fR can display up to 16 colours (8 \s-1ANSI\s0 colours plus
+\&\fBurxvt\fR can display up to 16 colours (8 \s-1ANSI\s0 colours plus
 high-intensity bold/blink versions of the same). Here is a list of the
 colours with their \fBrgb.txt\fR names.
 .TS
@@ -964,8 +964,8 @@
 always swapping the foreground/background colours. This is in contrast to
 \&\fIxterm\fR(1) where the colours are only swapped if they have not otherwise
 been specified. For example,
-.IP "\fBrxvt \-fg Black \-bg White \-rv\fR" 4
-.IX Item "rxvt -fg Black -bg White -rv"
+.IP "\fBurxvt \-fg Black \-bg White \-rv\fR" 4
+.IX Item "urxvt -fg Black -bg White -rv"
 would yield White on Black, while on \fIxterm\fR(1) it would yield Black
 on White.
 .SH "FREQUENTLY ASKED QUESTIONS (FAQ)"
@@ -979,13 +979,12 @@
 The terminal description used by rxvt-unicode is not as widely available
 as that for xterm, or even rxvt (for which the same problem often arises).
 .Sp
-The correct solution for this problem is to install the terminfo, this can be done
-like this:
+The correct solution for this problem is to install the terminfo, this can
+be done like this:
 .Sp
-.Vb 3
-\&   infocmp rxvt-unicode >rxvt.unicode.tic
-\&   scp rxvt-unicode.tic remotesystem:
-\&   ssh remotesystem tic rxvt-unicode.tic
+.Vb 2
+\&   REMOTE=remotesystem.domain
+\&   infocmp rxvt-unicode | ssh $REMOTE "cat >/tmp/ti && tic /tmp/ti"
 .Ve
 .Sp
 \&... or by installing rxvt-unicode normally on the remote system,
@@ -1002,6 +1001,9 @@
 .Vb 1
 \&   URxvt.termName: rxvt
 .Ve
+.Sp
+If you don't plan to use \fBrxvt\fR (quite common...) you could also replace
+the rxvt terminfo file with the rxvt-unicode one.
 .IP "How can I configure rxvt-unicode so that it looks similar to the original rxvt?" 4
 .IX Item "How can I configure rxvt-unicode so that it looks similar to the original rxvt?"
 Felix von Leitner says that these two lines, in your \fI.Xdefaults\fR, will make rxvt-unicode
@@ -1034,7 +1036,7 @@
 .Ve
 .Sp
 If this doesn't work, then maybe you use a \f(CW\*(C`LC_CTYPE\*(C'\fR specification not
-supported on your systems. Some systems have a \f(CW\*(C`locale\*(C'\fR comamnd which
+supported on your systems. Some systems have a \f(CW\*(C`locale\*(C'\fR command which
 displays this. If it displays sth. like:
 .Sp
 .Vb 1
@@ -1061,14 +1063,14 @@
 font. Often the result is fine, but sometimes the chosen font looks
 bad. Many fonts have totally strange characters that don't resemble the
 correct glyph at all, and rxvt-unicode lacks the artificial intelligence
-to detetc that a specific glyph is wrong: it has to believe the font that
+to detect that a specific glyph is wrong: it has to believe the font that
 the characters it contains indeed look correct.
 .Sp
 In that case, select a font of your taste and add it to the font list,
 e.g.:
 .Sp
 .Vb 1
-\&   rxvt -fn basefont,font2,font3...
+\&   urxvt -fn basefont,font2,font3...
 .Ve
 .Sp
 When rxvt-unicode sees a character, it will first look at the base
@@ -1077,7 +1079,7 @@
 search and use less resources within rxvt-unicode and the X\-server.
 .Sp
 The only limitation is that all the fonts must not be larger than the base
-font, as the base font defines the principial cell size, which must be the
+font, as the base font defines the principal cell size, which must be the
 same due to the way terminals work.
 .IP "Why do some chinese characters look so different than others?" 4
 .IX Item "Why do some chinese characters look so different than others?"
@@ -1085,7 +1087,7 @@
 rxvt-unicode does not know which language the text that is output
 is, as it only knows the unicode character codes. If rxvt-unicode
 first sees a japanese character, it might choose a japanese font for
-it. Subseqzuent japanese characters will take that font. Now, many chinese
+it. Subsequent japanese characters will take that font. Now, many chinese
 characters aren't represented in japanese fonts, so when the first
 non-japanese character comes up, rxvt-unicode will look for a chinese font
 \&\*(-- unfortunately at this point, it will still use the japanese font for
@@ -1204,7 +1206,7 @@
 .Ve
 .IP "My input method wants <some encoding> but I want \s-1UTF\-8\s0, what can I do?" 4
 .IX Item "My input method wants <some encoding> but I want UTF-8, what can I do?"
-You cna specify separate locales for the input method and the rest of the
+You can specify separate locales for the input method and the rest of the
 terminal, using the resource \f(CW\*(C`imlocale\*(C'\fR:
 .Sp
 .Vb 1
@@ -1226,7 +1228,7 @@
 Also, many people (me included) like large windows and even larger
 scrollback buffers: Without \f(CW\*(C`\-\-enable\-unicode3\*(C'\fR, rxvt-unicode will use
 6 bytes per screen cell. For a 160x?? window this amounts to almost a
-kilobyte per line. A scorllback buffer of 10000 lines will then (if full)
+kilobyte per line. A scrollback buffer of 10000 lines will then (if full)
 use 10 Megabytes of memory. With \f(CW\*(C`\-\-enable\-unicode3\*(C'\fR it gets worse, as
 rxvt-unicode then uses 8 bytes per screen cell.
 .IP "Can I speed up Xft rendering somehow?" 4
@@ -1240,7 +1242,7 @@
 Rxvt-unicode will use whatever you specify as a font. If it needs to
 fall back to it's default font search list it will prefer X11 core
 fonts, because they are small and fast, and then use Xft fonts. It has
-antialiaisng disabled for most of them, because the author thinks they
+antialiasing disabled for most of them, because the author thinks they
 look best that way.
 .Sp
 If you want antialiasing, you have to specify the fonts manually.
@@ -1250,7 +1252,7 @@
 some editors prematurely may leave the mouse in mouse report mode. I've
 heard that tcsh may use mouse reporting unless it otherwise specified. A
 quick check is to see if cut/paste works when the Alt or Shift keys are
-depressed. See \fIrxvt\fR\|(7)
+depressed. See \fIurxvt\fR\|(7)
 .IP "What's with this bold/blink stuff?" 4
 .IX Item "What's with this bold/blink stuff?"
 If no bold colour is set via \f(CW\*(C`colorBD:\*(C'\fR, bold will invert text using the
@@ -1318,16 +1320,16 @@
 .Vb 3
 \&   # use Backspace = ^H
 \&   $ stty erase ^H
-\&   $ rxvt
+\&   $ urxvt
 .Ve
 .Sp
 .Vb 3
 \&   # use Backspace = ^?
 \&   $ stty erase ^?
-\&   $ rxvt
+\&   $ urxvt
 .Ve
 .Sp
-Toggle with \*(L"ESC[36h\*(R" / \*(L"ESC[36l\*(R" as documented in \fIrxvt\fR\|(7).
+Toggle with \*(L"ESC[36h\*(R" / \*(L"ESC[36l\*(R" as documented in \fIurxvt\fR\|(7).
 .Sp
 For an existing rxvt\-unicode:
 .Sp
@@ -1366,7 +1368,7 @@
 use the `keysym' resource to alter the keystrings associated with keysym
 0xFF00 \- 0xFFFF (function, cursor keys, etc).
 .Sp
-Here's an example for a tn3270 session started using `rxvt \-name tn3270'
+Here's an example for a tn3270 session started using `urxvt \-name tn3270'
 .Sp
 .Vb 3
 \&   !#  ----- special uses ------:
@@ -1427,7 +1429,7 @@
 \&   F35 == Next
 .Ve
 .Sp
-Rather than have rxvt-unicode try to accomodate all the various possible keyboard
+Rather than have rxvt-unicode try to accommodate all the various possible keyboard
 mappings, it is better to use `xmodmap' to remap the keys as required for
 your particular machine.
 .IP "How do I distinguish if I'm running rxvt-unicode or a regular xterm? I need this to decide about setting colors etc." 4
@@ -1438,7 +1440,7 @@
 not to use color.
 .IP "How do I set the correct, full \s-1IP\s0 address for the \s-1DISPLAY\s0 variable?" 4
 .IX Item "How do I set the correct, full IP address for the DISPLAY variable?"
-If you've compiled rxvt-unicode with \s-1DISPLAY_IS_IP\s0 and ahve enabled
+If you've compiled rxvt-unicode with \s-1DISPLAY_IS_IP\s0 and have enabled
 insecure mode then it is possible to use the following shell script
 snippets to correctly set the display. If your version of rxvt-unicode
 wasn't also compiled with \s-1ESCZ_ANSWER\s0 (as assumed in these snippets) then
@@ -1469,11 +1471,11 @@
 the doc subdirectory and enter \f(CW\*(C`make alldoc\*(C'\fR.
 .SH "ENVIRONMENT"
 .IX Header "ENVIRONMENT"
-\&\fBrxvt\fR sets the environment variables \fB\s-1TERM\s0\fR, \fB\s-1COLORTERM\s0\fR
+\&\fBurxvt\fR sets the environment variables \fB\s-1TERM\s0\fR, \fB\s-1COLORTERM\s0\fR
 and \fB\s-1COLORFGBG\s0\fR. The environment variable \fB\s-1WINDOWID\s0\fR is set to the X
-window id number of the \fBrxvt\fR window and it also uses and
+window id number of the \fBurxvt\fR window and it also uses and
 sets the environment variable \fB\s-1DISPLAY\s0\fR to specify which display
-terminal to use. \fBrxvt\fR uses the environment variables
+terminal to use. \fBurxvt\fR uses the environment variables
 \&\fB\s-1RXVTPATH\s0\fR and \fB\s-1PATH\s0\fR to find \s-1XPM\s0 files.
 .SH "FILES"
 .IX Header "FILES"
@@ -1485,7 +1487,7 @@
 Color names.
 .SH "SEE ALSO"
 .IX Header "SEE ALSO"
-\&\fIrxvt\fR\|(7), \fIxterm\fR\|(1), \fIsh\fR\|(1), \fIresize\fR\|(1), X(1), \fIpty\fR\|(4), \fItty\fR\|(4), \fIutmp\fR\|(5)
+\&\fIurxvt\fR\|(7), \fIxterm\fR\|(1), \fIsh\fR\|(1), \fIresize\fR\|(1), X(1), \fIpty\fR\|(4), \fItty\fR\|(4), \fIutmp\fR\|(5)
 .SH "BUGS"
 .IX Header "BUGS"
 Check the \s-1BUGS\s0 file for an up-to-date list.
diff -urd rxvt-unicode-3.9/doc/rxvt.1.pod rxvt-unicode-3.9.my/doc/rxvt.1.pod
--- rxvt-unicode-3.9/doc/rxvt.1.pod	2004-08-24 17:46:18.000000000 +0200
+++ rxvt-unicode-3.9.my/doc/rxvt.1.pod	2004-08-28 11:11:37.000000000 +0200
@@ -26,7 +26,7 @@
 fine, though. A somewhat difficult case are left-to-right scripts, such
 as hebrew: B<rxvt-unicode> adopts the view that bidirectional algorithms
 belong into the application, not the terminal emulator (too many things --
-such as cursor-movement while editing -- break othwerwise), but that might
+such as cursor-movement while editing -- break otherwise), but that might
 change.
 
 If you are looking for a terminal that supports more exotic scripts, let
@@ -647,7 +647,7 @@
 
 =item B<scrollWithBuffer:> I<boolean>
 
-B<True>: scroll with scrollback buffer when tty recieves new lines (and
+B<True>: scroll with scrollback buffer when tty receives new lines (and
 B<scrollTtyOutput> is False); option B<+sw>. B<False>: do not scroll
 with scrollback buffer when tty recieves new lines; option B<-sw>.
 
@@ -760,7 +760,7 @@
 
 Enables "insecure" mode. Rxvt-unicode offers some escape sequences that
 echo arbitrary strings like the icon name or the locale. This could be
-abused if somebody gets 8-bit-clean access to your display, wether
+abused if somebody gets 8-bit-clean access to your display, whether
 throuh a mail client displaying mail bodies unfiltered or though
 write(1). Therefore, these sequences are disabled by default. (Note
 that other terminals, including xterm, have these sequences
@@ -901,7 +901,7 @@
 invoke it's usual function but instead will insert the corresponding
 keycap symbol. The symbol will only be entered when the key has been
 released, otherwise pressing e.g. C<Shift> would enter the symbol for
-C<ISO Level 2 Switch>, although your intention might have beenm to enter a
+C<ISO Level 2 Switch>, although your intention might have been to enter a
 reverse tab (Shift-Tab).
 
 =item 5.3: Screen-selection entry method
@@ -999,13 +999,12 @@
 The terminal description used by rxvt-unicode is not as widely available
 as that for xterm, or even rxvt (for which the same problem often arises).
 
-The correct solution for this problem is to install the terminfo, this can be done
-like this:
+The correct solution for this problem is to install the terminfo, this can
+be done like this:
+
+   REMOTE=remotesystem.domain
+   infocmp rxvt-unicode | ssh $REMOTE "cat >/tmp/ti && tic /tmp/ti"
 
-   infocmp rxvt-unicode >rxvt.unicode.tic
-   scp rxvt-unicode.tic remotesystem:
-   ssh remotesystem tic rxvt-unicode.tic
- 
 ... or by installing rxvt-unicode normally on the remote system,
 
 If you cannot or do not want to do this, then you can simply set
@@ -1019,6 +1018,9 @@
 
    URxvt.termName: rxvt
 
+If you don't plan to use B<rxvt> (quite common...) you could also replace
+the rxvt terminfo file with the rxvt-unicode one.
+
 =item How can I configure rxvt-unicode so that it looks similar to the original rxvt?
 
 Felix von Leitner says that these two lines, in your F<.Xdefaults>, will make rxvt-unicode
@@ -1046,7 +1048,7 @@
   printf '\e]701;%s\007' "$LC_CTYPE"
 
 If this doesn't work, then maybe you use a C<LC_CTYPE> specification not
-supported on your systems. Some systems have a C<locale> comamnd which
+supported on your systems. Some systems have a C<locale> command which
 displays this. If it displays sth. like:
 
   locale: Cannot set LC_CTYPE to default locale: ...
@@ -1070,7 +1072,7 @@
 font. Often the result is fine, but sometimes the chosen font looks
 bad. Many fonts have totally strange characters that don't resemble the
 correct glyph at all, and rxvt-unicode lacks the artificial intelligence
-to detetc that a specific glyph is wrong: it has to believe the font that
+to detect that a specific glyph is wrong: it has to believe the font that
 the characters it contains indeed look correct.
 
 In that case, select a font of your taste and add it to the font list,
@@ -1084,7 +1086,7 @@
 search and use less resources within rxvt-unicode and the X-server.
 
 The only limitation is that all the fonts must not be larger than the base
-font, as the base font defines the principial cell size, which must be the
+font, as the base font defines the principal cell size, which must be the
 same due to the way terminals work.
 
 =item Why do some chinese characters look so different than others?
@@ -1093,7 +1095,7 @@
 rxvt-unicode does not know which language the text that is output
 is, as it only knows the unicode character codes. If rxvt-unicode
 first sees a japanese character, it might choose a japanese font for
-it. Subseqzuent japanese characters will take that font. Now, many chinese
+it. Subsequent japanese characters will take that font. Now, many chinese
 characters aren't represented in japanese fonts, so when the first
 non-japanese character comes up, rxvt-unicode will look for a chinese font
 -- unfortunately at this point, it will still use the japanese font for
@@ -1207,7 +1209,7 @@
 
 =item My input method wants <some encoding> but I want UTF-8, what can I do?
 
-You cna specify separate locales for the input method and the rest of the
+You can specify separate locales for the input method and the rest of the
 terminal, using the resource C<imlocale>:
 
    URxvt*imlocale: ja_JP.EUC-JP
@@ -1228,7 +1230,7 @@
 Also, many people (me included) like large windows and even larger
 scrollback buffers: Without C<--enable-unicode3>, rxvt-unicode will use
 6 bytes per screen cell. For a 160x?? window this amounts to almost a
-kilobyte per line. A scorllback buffer of 10000 lines will then (if full)
+kilobyte per line. A scrollback buffer of 10000 lines will then (if full)
 use 10 Megabytes of memory. With C<--enable-unicode3> it gets worse, as
 rxvt-unicode then uses 8 bytes per screen cell.
 
@@ -1244,7 +1246,7 @@
 Rxvt-unicode will use whatever you specify as a font. If it needs to
 fall back to it's default font search list it will prefer X11 core
 fonts, because they are small and fast, and then use Xft fonts. It has
-antialiaisng disabled for most of them, because the author thinks they
+antialiasing disabled for most of them, because the author thinks they
 look best that way.
 
 If you want antialiasing, you have to specify the fonts manually.
@@ -1415,7 +1417,7 @@
    F33 == End
    F35 == Next
 
-Rather than have rxvt-unicode try to accomodate all the various possible keyboard
+Rather than have rxvt-unicode try to accommodate all the various possible keyboard
 mappings, it is better to use `xmodmap' to remap the keys as required for
 your particular machine.
 
@@ -1429,7 +1431,7 @@
 
 =item How do I set the correct, full IP address for the DISPLAY variable?
 
-If you've compiled rxvt-unicode with DISPLAY_IS_IP and ahve enabled
+If you've compiled rxvt-unicode with DISPLAY_IS_IP and have enabled
 insecure mode then it is possible to use the following shell script
 snippets to correctly set the display. If your version of rxvt-unicode
 wasn't also compiled with ESCZ_ANSWER (as assumed in these snippets) then
diff -urd rxvt-unicode-3.9/doc/rxvt.1.txt rxvt-unicode-3.9.my/doc/rxvt.1.txt
--- rxvt-unicode-3.9/doc/rxvt.1.txt	2004-08-25 04:35:55.000000000 +0200
+++ rxvt-unicode-3.9.my/doc/rxvt.1.txt	2004-08-28 11:11:37.000000000 +0200
@@ -3,7 +3,7 @@
     system)
 
 SYNOPSIS
-    rxvt [options] [-e command [ args ]]
+    urxvt [options] [-e command [ args ]]
 
 DESCRIPTION
     rxvt-unicode, version 3.8, is a colour vt102 terminal emulator intended
@@ -24,7 +24,7 @@
     scripts, such as hebrew: rxvt-unicode adopts the view that bidirectional
     algorithms belong into the application, not the terminal emulator (too
     many things -- such as cursor-movement while editing -- break
-    othwerwise), but that might change.
+    otherwise), but that might change.
 
     If you are looking for a terminal that supports more exotic scripts, let
     me recommend "mlterm", which is a very userfriendly, lean and clean
@@ -48,28 +48,28 @@
     without most of it's features to get a lean binary. It also comes with a
     client/daemon pair that lets you open any number of terminal windows
     from within a single process, which makes startup time very fast and
-    drastically reduces memory usage. See rxvtd(1) (daemon) and rxvtc(1)
+    drastically reduces memory usage. See urxvtd(1) (daemon) and urxvtc(1)
     (client).
 
     It also makes technical information about escape sequences (which have
-    been extended) easier accessible: see rxvt(7) for technical reference
+    been extended) easier accessible: see urxvt(7) for technical reference
     documentation (escape sequences etc.) and the FAQ section at the end of
     this document.
 
 OPTIONS
-    The rxvt options (mostly a subset of *xterm*'s) are listed below. In
+    The urxvt options (mostly a subset of *xterm*'s) are listed below. In
     keeping with the smaller-is-better philosophy, options may be eliminated
     or default values chosen at compile-time, so options and defaults listed
-    may not accurately reflect the version installed on your system. `rxvt
+    may not accurately reflect the version installed on your system. `urxvt
     -h' gives a list of major compile-time options on the *Options* line.
     Option descriptions may be prefixed with which compile option each is
     dependent upon. e.g. `Compile *XIM*:' requires *XIM* on the *Options*
-    line. Note: `rxvt -help' gives a list of all command-line options
+    line. Note: `urxvt -help' gives a list of all command-line options
     compiled into your version.
 
-    Note that rxvt permits the resource name to be used as a long-option
+    Note that urxvt permits the resource name to be used as a long-option
     (--/++ option) so the potential command-line options are far greater
-    than those listed. For example: `rxvt --loginShell --color1 Orange'.
+    than those listed. For example: `urxvt --loginShell --color1 Orange'.
 
     The following options are available:
 
@@ -231,7 +231,7 @@
         resource termName.
 
     -e *command [arguments]*
-        Run the command with its command-line arguments in the rxvt window;
+        Run the command with its command-line arguments in the urxvt window;
         also sets the window title and icon name to be the basename of the
         program being executed if neither *-title* (*-T*) nor *-n* are given
         on the command line. If this option is used, it must be the last on
@@ -287,26 +287,26 @@
         *WM_COMMAND* in some window managers.
 
 RESOURCES (available also as long-options)
-    Note: `rxvt --help' gives a list of all resources (long options)
+    Note: `urxvt --help' gives a list of all resources (long options)
     compiled into your version.
 
-    There are two different methods that rxvt can use to get the Xresource
+    There are two different methods that urxvt can use to get the Xresource
     data: using the X libraries (Xrm*-functions) or internal Xresources
-    reader (~/.Xdefaults). For the first method (ie. rxvt -h lists
+    reader (~/.Xdefaults). For the first method (ie. urxvt -h lists
     XGetDefaults), you can set and change the resources using X11 tools like
     xset. Many distribution do also load settings from the ~/.Xresources
     file when X starts.
 
-    If compiled with internal Xresources support (i.e. rxvt -h lists
-    .Xdefaults) then rxvt accepts application defaults set in
+    If compiled with internal Xresources support (i.e. urxvt -h lists
+    .Xdefaults) then urxvt accepts application defaults set in
     XAPPLOADDIR/URxvt (compile-time defined: usually
     /usr/lib/X11/app-defaults/URxvt) and resources set in ~/.Xdefaults, or
     ~/.Xresources if ~/.Xdefaults does not exist. Note that when reading X
-    resources, rxvt recognizes two class names: XTerm and URxvt. The class
-    name Rxvt allows resources common to both rxvt and the original *rxvt*
+    resources, urxvt recognizes two class names: XTerm and URxvt. The class
+    name Rxvt allows resources common to both urxvt and the original *rxvt*
     to be easily configured, while the class name URxvt allows resources
-    unique to rxvt, notably colours and key-handling, to be shared between
-    different rxvt configurations. If no resources are specified, suitable
+    unique to urxvt, notably colours and key-handling, to be shared between
+    different urxvt configurations. If no resources are specified, suitable
     defaults will be used. Command-line arguments can be used to override
     resource settings. The following resources are allowed:
 
@@ -332,7 +332,7 @@
         section.
 
         Colours higher than 15 cannot be set using resources (yet), but can
-        be changed using an escape command (see rxvt(7)).
+        be changed using an escape command (see urxvt(7)).
 
         Colours 16-79 form a standard 4x4x4 colour cube (the same as xterm
         with 88 colour support). Colours 80-87 are evenly spaces grey steps.
@@ -546,7 +546,7 @@
         do not scroll to bottom when tty receives output; option +si.
 
     scrollWithBuffer: *boolean*
-        True: scroll with scrollback buffer when tty recieves new lines (and
+        True: scroll with scrollback buffer when tty receives new lines (and
         scrollTtyOutput is False); option +sw. False: do not scroll with
         scrollback buffer when tty recieves new lines; option -sw.
 
@@ -640,7 +640,7 @@
         Enables "insecure" mode. Rxvt-unicode offers some escape sequences
         that echo arbitrary strings like the icon name or the locale. This
         could be abused if somebody gets 8-bit-clean access to your display,
-        wether throuh a mail client displaying mail bodies unfiltered or
+        whether throuh a mail client displaying mail bodies unfiltered or
         though write(1). Therefore, these sequences are disabled by default.
         (Note that other terminals, including xterm, have these sequences
         enabled by default). You can enable them by setting this boolean
@@ -676,9 +676,9 @@
         is only available when compiled with KEYSYM_RESOURCE.
 
 THE SCROLLBAR
-    Lines of text that scroll off the top of the rxvt window (resource:
+    Lines of text that scroll off the top of the urxvt window (resource:
     saveLines) and can be scrolled back using the scrollbar or by
-    keystrokes. The normal rxvt scrollbar has arrows and its behaviour is
+    keystrokes. The normal urxvt scrollbar has arrows and its behaviour is
     fairly intuitive. The xterm-scrollbar is without arrows and its
     behaviour mimics that of *xterm*
 
@@ -709,7 +709,7 @@
 
     Insertion:
         Pressing and releasing the Middle mouse button (or Shift-Insert) in
-        an rxvt window causes the current text selection to be inserted as
+        an urxvt window causes the current text selection to be inserted as
         if it had been typed on the keyboard.
 
 CHANGING FONTS
@@ -757,7 +757,7 @@
         corresponding keycap symbol. The symbol will only be entered when
         the key has been released, otherwise pressing e.g. "Shift" would
         enter the symbol for "ISO Level 2 Switch", although your intention
-        might have beenm to enter a reverse tab (Shift-Tab).
+        might have been to enter a reverse tab (Shift-Tab).
 
     5.3: Screen-selection entry method
         While this is implemented already (it's basically the selection
@@ -784,12 +784,12 @@
     both scenario A and B of ISO 14755, including part 5.2.
 
 LOGIN STAMP
-    rxvt tries to write an entry into the *utmp*(5) file so that it can be
+    urxvt tries to write an entry into the *utmp*(5) file so that it can be
     seen via the *who(1)* command, and can accept messages. To allow this
-    feature, rxvt must be installed setuid root on some systems.
+    feature, urxvt must be installed setuid root on some systems.
 
 COLORS AND GRAPHICS
-    In addition to the default foreground and background colours, rxvt can
+    In addition to the default foreground and background colours, urxvt can
     display up to 16 colours (8 ANSI colours plus high-intensity bold/blink
     versions of the same). Here is a list of the colours with their rgb.txt
     names.
@@ -823,7 +823,7 @@
     *xterm*(1) where the colours are only swapped if they have not otherwise
     been specified. For example,
 
-    rxvt -fg Black -bg White -rv
+    urxvt -fg Black -bg White -rv
         would yield White on Black, while on *xterm*(1) it would yield Black
         on White.
 
@@ -840,10 +840,9 @@
         The correct solution for this problem is to install the terminfo,
         this can be done like this:
 
-           infocmp rxvt-unicode >rxvt.unicode.tic
-           scp rxvt-unicode.tic remotesystem:
-           ssh remotesystem tic rxvt-unicode.tic
- 
+           REMOTE=remotesystem.domain
+           infocmp rxvt-unicode | ssh $REMOTE "cat >/tmp/ti && tic /tmp/ti"
+
         ... or by installing rxvt-unicode normally on the remote system,
 
         If you cannot or do not want to do this, then you can simply set
@@ -858,6 +857,9 @@
 
            URxvt.termName: rxvt
 
+        If you don't plan to use rxvt (quite common...) you could also
+        replace the rxvt terminfo file with the rxvt-unicode one.
+
     How can I configure rxvt-unicode so that it looks similar to the
     original rxvt?
         Felix von Leitner says that these two lines, in your .Xdefaults,
@@ -886,7 +888,7 @@
           printf '\e]701;%s\007' "$LC_CTYPE"
 
         If this doesn't work, then maybe you use a "LC_CTYPE" specification
-        not supported on your systems. Some systems have a "locale" comamnd
+        not supported on your systems. Some systems have a "locale" command
         which displays this. If it displays sth. like:
 
           locale: Cannot set LC_CTYPE to default locale: ...
@@ -908,14 +910,14 @@
         Often the result is fine, but sometimes the chosen font looks bad.
         Many fonts have totally strange characters that don't resemble the
         correct glyph at all, and rxvt-unicode lacks the artificial
-        intelligence to detetc that a specific glyph is wrong: it has to
+        intelligence to detect that a specific glyph is wrong: it has to
         believe the font that the characters it contains indeed look
         correct.
 
         In that case, select a font of your taste and add it to the font
         list, e.g.:
 
-           rxvt -fn basefont,font2,font3...
+           urxvt -fn basefont,font2,font3...
 
         When rxvt-unicode sees a character, it will first look at the base
         font. If the base font does not contain the character, it will go to
@@ -924,7 +926,7 @@
         X-server.
 
         The only limitation is that all the fonts must not be larger than
-        the base font, as the base font defines the principial cell size,
+        the base font, as the base font defines the principal cell size,
         which must be the same due to the way terminals work.
 
     Why do some chinese characters look so different than others?
@@ -932,7 +934,7 @@
         rxvt-unicode does not know which language the text that is output
         is, as it only knows the unicode character codes. If rxvt-unicode
         first sees a japanese character, it might choose a japanese font for
-        it. Subseqzuent japanese characters will take that font. Now, many
+        it. Subsequent japanese characters will take that font. Now, many
         chinese characters aren't represented in japanese fonts, so when the
         first non-japanese character comes up, rxvt-unicode will look for a
         chinese font -- unfortunately at this point, it will still use the
@@ -1043,7 +1045,7 @@
            URxvt*boldItalicFont:    xft:Bitstream Vera Sans Mono:bold:italic:autohint=true
 
     My input method wants <some encoding> but I want UTF-8, what can I do?
-        You cna specify separate locales for the input method and the rest
+        You can specify separate locales for the input method and the rest
         of the terminal, using the resource "imlocale":
 
            URxvt*imlocale: ja_JP.EUC-JP
@@ -1064,7 +1066,7 @@
         Also, many people (me included) like large windows and even larger
         scrollback buffers: Without "--enable-unicode3", rxvt-unicode will
         use 6 bytes per screen cell. For a 160x?? window this amounts to
-        almost a kilobyte per line. A scorllback buffer of 10000 lines will
+        almost a kilobyte per line. A scrollback buffer of 10000 lines will
         then (if full) use 10 Megabytes of memory. With "--enable-unicode3"
         it gets worse, as rxvt-unicode then uses 8 bytes per screen cell.
 
@@ -1078,7 +1080,7 @@
         Rxvt-unicode will use whatever you specify as a font. If it needs to
         fall back to it's default font search list it will prefer X11 core
         fonts, because they are small and fast, and then use Xft fonts. It
-        has antialiaisng disabled for most of them, because the author
+        has antialiasing disabled for most of them, because the author
         thinks they look best that way.
 
         If you want antialiasing, you have to specify the fonts manually.
@@ -1088,7 +1090,7 @@
         some editors prematurely may leave the mouse in mouse report mode.
         I've heard that tcsh may use mouse reporting unless it otherwise
         specified. A quick check is to see if cut/paste works when the Alt
-        or Shift keys are depressed. See rxvt(7)
+        or Shift keys are depressed. See urxvt(7)
 
     What's with this bold/blink stuff?
         If no bold colour is set via "colorBD:", bold will invert text using
@@ -1152,13 +1154,13 @@
 
            # use Backspace = ^H
            $ stty erase ^H
-           $ rxvt
+           $ urxvt
 
            # use Backspace = ^?
            $ stty erase ^?
-           $ rxvt
+           $ urxvt
 
-        Toggle with "ESC[36h" / "ESC[36l" as documented in rxvt(7).
+        Toggle with "ESC[36h" / "ESC[36l" as documented in urxvt(7).
 
         For an existing rxvt-unicode:
 
@@ -1193,7 +1195,7 @@
         option you can use the `keysym' resource to alter the keystrings
         associated with keysym 0xFF00 - 0xFFFF (function, cursor keys, etc).
 
-        Here's an example for a tn3270 session started using `rxvt -name
+        Here's an example for a tn3270 session started using `urxvt -name
         tn3270'
 
            !#  ----- special uses ------:
@@ -1245,7 +1247,7 @@
            F33 == End
            F35 == Next
 
-        Rather than have rxvt-unicode try to accomodate all the various
+        Rather than have rxvt-unicode try to accommodate all the various
         possible keyboard mappings, it is better to use `xmodmap' to remap
         the keys as required for your particular machine.
 
@@ -1257,7 +1259,7 @@
         whether or not to use color.
 
     How do I set the correct, full IP address for the DISPLAY variable?
-        If you've compiled rxvt-unicode with DISPLAY_IS_IP and ahve enabled
+        If you've compiled rxvt-unicode with DISPLAY_IS_IP and have enabled
         insecure mode then it is possible to use the following shell script
         snippets to correctly set the display. If your version of
         rxvt-unicode wasn't also compiled with ESCZ_ANSWER (as assumed in
@@ -1286,10 +1288,10 @@
         Then go to the doc subdirectory and enter "make alldoc".
 
 ENVIRONMENT
-    rxvt sets the environment variables TERM, COLORTERM and COLORFGBG. The
+    urxvt sets the environment variables TERM, COLORTERM and COLORFGBG. The
     environment variable WINDOWID is set to the X window id number of the
-    rxvt window and it also uses and sets the environment variable DISPLAY
-    to specify which display terminal to use. rxvt uses the environment
+    urxvt window and it also uses and sets the environment variable DISPLAY
+    to specify which display terminal to use. urxvt uses the environment
     variables RXVTPATH and PATH to find XPM files.
 
 FILES
@@ -1300,7 +1302,7 @@
         Color names.
 
 SEE ALSO
-    rxvt(7), xterm(1), sh(1), resize(1), X(1), pty(4), tty(4), utmp(5)
+    urxvt(7), xterm(1), sh(1), resize(1), X(1), pty(4), tty(4), utmp(5)
 
 BUGS
     Check the BUGS file for an up-to-date list.
Nur in rxvt-unicode-3.9/doc: rxvt.7.html.
diff -urd rxvt-unicode-3.9/doc/rxvt.7.man rxvt-unicode-3.9.my/doc/rxvt.7.man
--- rxvt-unicode-3.9/doc/rxvt.7.man	2004-08-25 05:54:53.000000000 +0200
+++ rxvt-unicode-3.9.my/doc/rxvt.7.man	2004-08-28 11:11:37.000000000 +0200
@@ -129,7 +129,7 @@
 .\" ========================================================================
 .\"
 .IX Title "rxvt 7"
-.TH rxvt 7 "2004-08-25" "perl v5.8.4" "RXVT-UNICODE"
+.TH rxvt 7 "2004-08-28" "perl v5.8.4" "RXVT-UNICODE"
 .SH "NAME"
 RXVT TECHNICAL REFERENCE \- command sequences and background information
 .SH "SYNOPSIS"
@@ -889,7 +889,7 @@
 Ps = 49	Change default background colour to Pt rxvt compile-time option
 Ps = 50	Set fontset to Pt, with the following special values of Pt (rxvt) #+n change up n #-n change down n if n is missing of 0, a value of 1 is used empty change to font0 n change to font n
 Ps = 55	Log all scrollback buffer and all of screen to Pt
-Ps = 701	Change current locale to Pt, or, if Pt is ?, return the current locale (rxvt extension)
+Ps = 701	Change current locale to Pt, or, if Pt is ?, return the current locale (urxvt extension)
 Ps = 703	Menubar command Pt rxvt compile-time option (rxvt-unicode extension)
 Ps = 704	Change colour of italic characters to Pt
 Ps = 705	Change background pixmap tint colour to Pt
@@ -1601,7 +1601,7 @@
 disable this.
 .IP "\-\-enable\-iso14755" 4
 .IX Item "--enable-iso14755"
-Enable extended \s-1ISO\s0 14755 support (see \fIrxvt\fR\|(1), or
+Enable extended \s-1ISO\s0 14755 support (see \fIurxvt\fR\|(1), or
 \&\fIdoc/rxvt.1.txt\fR). Basic support (section 5.1) is enabled by
 \&\f(CW\*(C`\-\-enable\-frills\*(C'\fR, while support for 5.2, 5.3 and 5.4 is enabled with
 this switch.
diff -urd rxvt-unicode-3.9/doc/rxvt.7.txt rxvt-unicode-3.9.my/doc/rxvt.7.txt
--- rxvt-unicode-3.9/doc/rxvt.7.txt	2004-08-25 05:54:56.000000000 +0200
+++ rxvt-unicode-3.9.my/doc/rxvt.7.txt	2004-08-28 11:11:37.000000000 +0200
@@ -476,7 +476,7 @@
            Ps = 49         Change default background colour to Pt rxvt compile-time option 
            Ps = 50         Set fontset to Pt, with the following special values of Pt (rxvt) #+n change up n #-n change down n if n is missing of 0, a value of 1 is used empty change to font0 n change to font n 
            Ps = 55         Log all scrollback buffer and all of screen to Pt 
-           Ps = 701        Change current locale to Pt, or, if Pt is ?, return the current locale (rxvt extension) 
+           Ps = 701        Change current locale to Pt, or, if Pt is ?, return the current locale (urxvt extension) 
            Ps = 703        Menubar command Pt rxvt compile-time option (rxvt-unicode extension) 
            Ps = 704        Change colour of italic characters to Pt 
            Ps = 705        Change background pixmap tint colour to Pt 
@@ -1125,7 +1125,7 @@
         want to disable this.
 
     --enable-iso14755
-        Enable extended ISO 14755 support (see rxvt(1), or doc/rxvt.1.txt).
+        Enable extended ISO 14755 support (see urxvt(1), or doc/rxvt.1.txt).
         Basic support (section 5.1) is enabled by "--enable-frills", while
         support for 5.2, 5.3 and 5.4 is enabled with this switch.
 
Nur in rxvt-unicode-3.9.my/doc: rxvt.tbl.
diff -urd rxvt-unicode-3.9/README.configure rxvt-unicode-3.9.my/README.configure
--- rxvt-unicode-3.9/README.configure	2004-08-25 05:54:57.000000000 +0200
+++ rxvt-unicode-3.9.my/README.configure	2004-08-28 11:11:55.000000000 +0200
@@ -170,7 +170,7 @@
         want to disable this.
 
     --enable-iso14755
-        Enable extended ISO 14755 support (see rxvt(1), or doc/rxvt.1.txt).
+        Enable extended ISO 14755 support (see urxvt(1), or doc/rxvt.1.txt).
         Basic support (section 5.1) is enabled by "--enable-frills", while
         support for 5.2, 5.3 and 5.4 is enabled with this switch.
 
